src/libraries/Common/src/System/IO/MemoryMappedFileInternal.Windows.cs

```diff
@@ -0,0 +1,280 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file used to be the Windows partial class for `MemoryMappedFile`, but since a big chunk of the code in that file needed to be used by the new methods, I decided to move the shared code to a new Common class called `MemoryMappedFileInternal` to avoid code duplication.